viewer/src/glsl/base/updateFragmentColor.glsl

```diff
@@ -2,6 +2,12 @@
 #pragma glslify: hsv2rgb = require('../color/hsv2rgb.glsl')
 #pragma glslify: packIntToColor = require('../color/packIntToColor.glsl')
 
+uniform float renderNormals;
```

Comment:
Send as argument from each shader, but as a 'renderType'. Ok if this is done in separate PR.